### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by 2can**
**I want to merge this PR because: ***
It creates docs directory with all commands usage

## Related issues

- https://github.com/saleor/saleor-cli/issues/283 

## Steps to test feature

- 

## I have:

- [x] Tested it locally and it doesn't break existing features
- [x] Added documentation if public changes are introduced
- [ ] Added tests for my code

### CODE REVIEW

**Review by zaiste — APPROVED** [review]

File: docs/app.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+## saleor app
```
Comment (lkostrowski) [inline docs/app.md:1]: Lets name file to something like `saleor-app-command`, so its clear when docs are about commands

File: docs/app.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      --json     Output the data as JSON                               [boolean]
+  -V, --version  Show version number                                   [boolean]
+  -h, --help     Show help                                             [boolean]
+```
+
+Example usage in batch mode
+
+```
+saleor app list
+```
```
Comment (lkostrowski) [inline docs/app.md:39]: How this is useful?

File: docs/app.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+      --dependencies, --deps                           [boolean] [default: true]
+  -V, --version               Show version number                      [boolean]
+  -h, --help                  Show help                                [boolean]
+```
+
+Example usage in batch mode
+
+```
+saleor app create my-app-name
+```
```
Comment (lkostrowski) [inline docs/app.md:84]: If command is non-interactive, example doesnt make sense. I'd remove it

File: docs/app.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+      --name     The application name for installation in the Dashboard [string]
+  -V, --version  Show version number                                   [boolean]
+  -h, --help     Show help                                             [boolean]
+```
+
+Example usage in batch mode
+
+```
+saleor app tunnel 5000
+```
```
Comment (lkostrowski) [inline docs/app.md:107]: IMO This is not example of batch, because tunnel is not asking interactively about port.

**Review by lkostrowski — CHANGES_REQUESTED** [review]
Please use batch mode examples only for interactive commands that can be made non-interactive.
TBH we can make one section, that explains all "flags" can be passed in single command, we don't have to repeat it

@peelar what do you think?

File: docs/app.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+Options:
+      --json           Output the data as JSON                         [boolean]
+      --via-dashboard                                 [boolean] [default: false]
+      --app-name       Application name                                 [string]
+      --manifest-URL   Application Manifest URL                         [string]
+  -V, --version        Show version number                             [boolean]
+  -h, --help           Show help                                       [boolean]
+```
+
+Example usage in batch mode
```
Comment (peelar) [inline docs/app.md:57]: What does "batch mode" mean? With several options? If so, I would change the wording here or even remove it.

Comment (zaiste) [reply docs/app.md:57]: @peelar *batch mode* means that arguments are passed from the invocation, and not interactively

File: docs/app.md (hunk 2 above)
Comment (2can) [reply docs/app.md]: removed

File: docs/app.md (hunk 4 above)
Comment (2can) [reply docs/app.md]: removed

File: docs/app.md (hunk 3 above)
Comment (2can) [reply docs/app.md]: removed

File: docs/app.md (hunk 1 above)
Comment (2can) [reply docs/app.md]: updated

File: docs/app.md (hunk 5 above)
Comment (2can) [reply docs/app.md:51]: @peelar is it OK? Do you think it should be re-phrased?

File: docs/app.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+      --json     Output the data as JSON                               [boolean]
+  -V, --version  Show version number                                   [boolean]
+  -h, --help     Show help                                             [boolean]
+```
+
+Example usage in batch mode
+
+```
+<!-- TODO -->
+```
```
Comment (lkostrowski) [inline docs/app.md:128]: Why do you leave this TODO?

File: docs/backup.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+  saleor backup restore [from]          Restore a specific backup
+
+Options:
+      --json     Output the data as JSON                               [boolean]
+  -V, --version  Show version number                                   [boolean]
+  -h, --help     Show help                                             [boolean]
+
+```
+
+ saleor backup list [key|environment]  List backups of the environment
```
Comment (lkostrowski) [inline docs/backup.md:19]: fix indent

**Review by lkostrowski — CHANGES_REQUESTED** [review]

File: docs/app.md (hunk 5 above)
Comment (lkostrowski) [reply docs/app.md:51]: @2can @peelar lets have a section in `readme.md` `# Batch Mode (non-interactive)` that will explain in few words that commands can be run interactive and non-interactive (batch mode), when batch is good (programmatic usage) and maybe link to some Wikipedia or something

Comment (2can) [reply docs/app.md:51]: OK, so let's use "non-interactive"  instead of "batch"

File: docs/app.md (hunk 6 above)
Comment (2can) [reply docs/app.md]: Removed. Created issue for that https://github.com/saleor/saleor-cli/issues/352

**Review by lkostrowski — APPROVED** [review]

**Pull request merged**